src/PackageUtilities.js

```diff
@@ -218,4 +219,147 @@ export default class PackageUtilities {
       async.parallelLimit(batch.map(makeTask), concurrency, cb);
     }), callback);
   }
+
+
+  /**
+   * Symlink all packages to the packages/node_modules directory
+   * Symlink package binaries to dependent packages' node_modules/.bin directory
+   * @param {Array.<Package>} packages
+   * @param {Object} packageGraph
+   * @param {Object} logger
+   * @param {Function} callback
+   */
+  static symlinkPackages(packages, packageGraph, logger, callback) {
```

Comment:
I love this refactoring!

Reply:
Awesome! Will add the feedback tomorrow :) 👍